helm-org.el

```diff
@@ -58,9 +58,11 @@ NOTE: This will be slow on large org buffers."
   (org-show-entry))
 
 (cl-defun helm-source-org-headings-for-files (filenames
-                                              &optional (min-depth 1) (max-depth 8))
+                                              &optional (min-depth 1) (max-depth 8) (parents nil))
   (helm-build-sync-source "Org Headings"
-    :candidates (helm-org-get-candidates filenames min-depth max-depth)
+    :candidates (helm-org-get-candidates filenames min-depth max-depth parents)
+    :candidate-transformer (if parents 'reverse nil)
+    :after-init-hook 'helm-follow-mode
```

Comment:
That works, but it still has the problem I mentioned: the `follow` happens before the `preselect`, which causes the point to briefly jump to the top of the buffer before returning to the current heading, which is a bit distracting and...unpolished.    :)

The docstring for `after-init-hook` says that it runs after the Helm buffer is created, which sounds like it should allow activating of `helm-follow-mode` after the `preselect` runs, but I haven't been able to find a way to activate `helm-follow-mode` from the `after-init-hook`.  Is there a way to do this?